frontend/src/bundles/video_editor/components/timeline/subcomponents/item.tsx

```diff
@@ -0,0 +1,41 @@
+import { type Span } from 'dnd-timeline';
+import { useItem } from 'dnd-timeline';
+
+interface ItemProperties {
```

Comment:
You should use Type and call it just Properties

Reply:
Done. Thank you. 
Could you tell me why is it better type than interface?

Reply:
interface has a more OOP approach so, I'd say it's better for classes. Also to be consistent with the whole project

Reply:
It doesn't mean we cannot use types in an OOP paradigm, is more what you need, preferences, and as in this case consistency

Reply:
Here you can find real differences : [https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces](type vs interface)

Reply:
Thank you. Its clear now